docs/system-admin-guide/users-permissions/roles-permissions/README.md

```diff
@@ -125,11 +125,15 @@ To delete an existing role click on the **delete icon** next to a role in the li
 
 No, only Administrators can delete other users.
 
+### Can a user with "Edit users" global permission change administrators attributes?
+
+No, only Administrators can update other Administrators attributes like name or email. This is to prevent the possibility of a user with "Edit users" global permission to impersonate an Administrator by changing the email address to match theirs.
```

Comment:
```suggestion
No, only Administrators can update other Administrators attributes like name or email. This is to prevent the possibility of a user with "Edit users" global permission impersonating an Administrator by changing the email address to match theirs.
```